### PR TITLE
add custom search + embed planeteclipse.org posts

### PR DESCRIPTION
**Pull request by chrisguindon**
Fix https://github.com/EclipseFdn/ecdtools.eclipse.org/issues/160
Fix https://github.com/EclipseFdn/ecdtools.eclipse.org/issues/157

Signed-off-by: Christopher Guindon <chris.guindon@eclipse-foundation.org>

### CODE REVIEW

**Review by ericpoirier — APPROVED** [review]

**Review by ericpoirier — APPROVED** [review]

File: js/src/EclipseBlogs.js [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+          const blogPosts = [];
+          items.forEach((el) => {
+            const item = {
+              title: '',
+              link: '',
+              summary: '',
+              date: '',
+              id: '',
+              author: '',
+              formatDate: function () {
```
Comment (ericpoirier) [inline js/src/EclipseBlogs.js:45]: Why do you have 2 nested functions here?

File: content/membership/join-us/_index.md [hunk 2]
```diff
@@ -5,4 +5,4 @@ hide_sidebar: true
 jumbotron_bg_class: "featured-jumbotron-sub-pages header-membership-bg-img"
 ---
 
-{{< page/join-us >}}
+{{< hubspot_contact_form portalId="5413615" formId="b2d1d804-1d76-4fe9-9647-03f478af1780" >}}
```
Comment (ericpoirier) [inline content/membership/join-us/_index.md:8]: I'm assuming you're adding this here since you're making changes to ECDtools?

**Review by ericpoirier — COMMENTED** [review]
Looks good but your PR has conflicts

File: js/src/EclipseBlogs.js (hunk 1 above)
Comment (chrisguindon) [reply js/src/EclipseBlogs.js]: See lambda section https://mustache.github.io/mustache.5.html

File: content/membership/join-us/_index.md (hunk 2 above)
Comment (chrisguindon) [reply content/membership/join-us/_index.md:8]: We should be using our shortcode for this.

Comment (chrisguindon) [reply content/membership/join-us/_index.md:8]: I don't see any conflicts?

**Pull request merged**